packages/libs/react-ui/src/components/Pagination/Pagination.stories.tsx

```diff
@@ -50,12 +70,19 @@ type Story = StoryObj<IPaginationProps>;
 
 export const Controlled: Story = {
   args: {
-    label: 'Label',
+    label: 'Pagination',
     totalPages: 10,
     visiblePageLimit: 3,
+    initialSelectedPage: 2,
   },
-  render: ({ totalPages, label, visiblePageLimit }) => {
-    const [page, setPage] = React.useState(1);
+  render: ({ totalPages, label, visiblePageLimit, initialSelectedPage }) => {
+    const validInitialSelectedPage =
+      initialSelectedPage &&
+      initialSelectedPage <= totalPages &&
+      initialSelectedPage > 0;
+    const [page, setPage] = React.useState(
+      validInitialSelectedPage ? initialSelectedPage : 1,
+    );
```

Comment:
Can this logic be removed from the story? I think the component should handle it internally